### PR TITLE
Add recipe for domd image

### PR DESCRIPTION
**Pull request by dsemenets**
Simple image includes:
- xen and u-boot packages
- disable wic image creation

### CODE REVIEW

File: recipes-core/image/image.bb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+SUMMARY = "A small image just capable of allowing a device to boot."
```
Comment (GrygiriiS) [inline recipes-core/image/image.bb]: if "image.bb" will become a build target - it looks not good and another name better to select

Comment (oleksiimoisieiev) [reply recipes-core/image/image.bb]: You can take 
https://github.com/aoscloud/meta-aos-rcar-gen4/blob/main/aos-rcar-gen4.yaml
as reference for image structure

**Review by oleksiimoisieiev — CHANGES_REQUESTED** [review]

File: recipes-core/image/image.bb (hunk 1 above)
Comment (dsemenets) [reply recipes-core/image/image.bb]: Ok

Comment (GrygiriiS) [reply recipes-core/image/image.bb]: rpi5-image-minimal-domd.bb

**Comment by GrygiriiS** [comment]
LGTM

**Comment by oleksiimoisieiev** [comment]
LGTM

**Pull request merged**